tldw_Server_API/tests/Metrics/test_telemetry_lifecycle.py

```
@@ -0,0 +1,258 @@
+        print("ALIGN_SECOND_METER", meter_provider2 is manager2.meter_provider)
+        print("REUSE_TRACE_GLOBAL", trace_provider2 is trace_provider1)
+        print("REUSE_METER_GLOBAL", meter_provider2 is meter_provider1)
+        """
+    )
+
+    env = os.environ.copy()
+    env["OTEL_SDK_DISABLED"] = "false"
+    env["OTEL_TRACES_EXPORTER"] = "console"
+    env["OTEL_METRICS_EXPORTER"] = "console"
+
+    result = subprocess.run(  # nosec B603
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)